archive/tar.go

```diff
@@ -32,7 +32,7 @@ import (
 
 	"github.com/containerd/containerd/log"
 	"github.com/containerd/containerd/pkg/userns"
-	"github.com/containerd/continuity/fs"
+	"continuity/fs"
```

Comment:
?

Reply:
This is just for testing. We also needed custom changes in continuity that are not checked in. So we tested by checking it in. If you think this is worth merging, we will rebase to master after the continuity changes are merged to master.